### ISSUE TITLE
Super buggy behaviour after upgrade to EXPO 36 

### ISSUE THREAD

**Issue by Hirbod** [CrowdLinker/react-native-pager#21]
Hi,

I just upgraded to EXPO SDK 36 which includes React Native 0.61.4 and react-native-reanimated 1.4.0 and react-native-gesture-handler 1.5.0.

After upgrading, the whole Pager is broken for me.
I've uploaded you a video.

https://www.youtube.com/watch?v=COA_LwrWfK4&

So the first image is always working, but any index > 1 will start to act super weird. They stay in the transitioned position until I touch the screen and move it by 1 pixel. If I chose to take an element which is out of my adjacentChildOffset, it just displays nothing until I touch and move for a pixel.

Is there any chance you could have a quick look into this? My schedule on this app is for friday and I spent a decent amount of time upgrading to SDK 36 and this is the last puzzle piece that is broken now for me :(

**Comment by Hirbod** [comment]
Ok, I managed to fix this myself FOR NOW.

I exported my Pager with

```
function arePropsEqual(prevProps, nextProps) {
    return prevProps.stories === nextProps.stories;
}

export default memo(Stories, arePropsEqual);
```

So removing memo(), I don't have these blanks anymore and my stuff works. But the transition stuff was still flickering there for a second until the image showed up. It was fixed after I changed the pageInterpolation. I had to add

```
    zIndex: (offset: Animated.Node<number>) => multiply(floor(offset), -1),

    opacity: {
        inputRange: [-2, -1, 0, 1, 2, 3, 4],
        outputRange: [0, 0, 1, 1, 1, 0, 0],
    },
```
as in your examples. I guess there were there for a reason, but I removed it, because that zindex stuff looked like useless for my case, but after that upgrade I got your point. I leave this open for now, because I am curious why `memo()` is breaking on this new version
